aiida/cmdline/commands/cmd_graph.py

```diff
@@ -15,20 +15,31 @@
 import click
 
 from aiida.cmdline.commands.cmd_verdi import verdi
-from aiida.cmdline.params import arguments
+from aiida.cmdline.params import arguments, options
 from aiida.cmdline.utils import decorators, echo
 
 
 @verdi.group('graph')
 def verdi_graph():
-    """
-    Create visual representations of part of the provenance graph.
-    Requires that `graphviz<https://graphviz.org/download>` be installed.
+    """Create visual representations of part of the provenance graph.
     """
 
 
 @verdi_graph.command('generate')
 @arguments.NODE('root_node')
+@click.option(
+    "-l",
```

Comment:
A nitpick, feel free to ignore. In `aiida-core` we are adhering to the convention to use single quotes for identifier strings, such as the keywords here and where possible in general. For "text"-like string, such as the help messages, which often contain single quotes, it is fine to use double quotes (whichever option leads to the least escaping characters).

Reply:
Yeh, it makes. I'll fix a few, but might not get all. FYI, there's an open issue on YAPf to do this google/yapf#399

Reply:
That'd be great indeed to have yapf do this. And don't worry, as I said, just a nitpick, great if you take a few along